src/org/opendatakit/briefcase/ui/reused/AggregateServerConnectionDialog.java

```diff
@@ -0,0 +1,68 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import java.awt.Cursor;
```

Comment:
Ideally, this class wouldn't have any imports from Swing. Remember: the view knows about the UI. The controller knows about business